lib/common/output_log.c

```diff
@@ -277,3 +291,11 @@ pcmk__mk_log_output(char **argv) {
 
     return retval;
 }
+
+void
+pcmk__output_set_log_level(pcmk__output_t *out, int log_level) {
+    private_data_t *priv = out->priv;
+
+    CRM_ASSERT(priv != NULL);
+    priv->log_level = log_level;
```

Comment:
It occurs to me that with these formatter-specific utility functions, we should do nothing if the format isn't what it's supposed to be, just in case we call it somewhere with a generic output object by mistake (don't want to clobber some other format's private data)

Reply:
There's no better way to do this than by comparing strings (there's compiler extensions like typeof, but...), and some care does need to be taken since the XML functions can be used by HTML functions, but added.